### DISCUSSION TITLE
Increase bar menu size

### DISCUSSION THREAD

**Discussion by fatalkam** [Megabit/Blazorise#2095] [Q&A] [answered]
I have a issue where the text is bumping into the toggle image. I also would like to increase the size just a little but it seems to be fixed. Is there a way to do this?

![image](https://user-images.githubusercontent.com/31685014/112347725-1bd6a680-8c95-11eb-9df4-d06a119f732c.png)

**Comment by stsrki** [comment, marked as answer]
Why don't you use CSS selector for the dropdown items and adjust the `text-overflow: ellipsis; `? I think that is the easiest solution.

https://www.w3schools.com/cssref/css3_pr_text-overflow.asp

**Reply by fatalkam** [reply]
Thank you for the reply. I have tired adding the style but it did not change anything. Is there no way to increase the max width?

                        <BarItem>
                            <BarDropdown>
                                <BarDropdownToggle>
                                    <BarIcon IconName="IconName.Dice" />
                                    CLLI
                                </BarDropdownToggle>
                                <BarDropdownMenu>
                                    <BarDropdown>
                                        <BarDropdownToggle Style="text-overflow: ellipsis;">
                                            Network Entity Format
                                        </BarDropdownToggle>
                                        <BarDropdownMenu>
                                            <BarDropdownItem To="/networkentityformatscan">
                                                Scan
                                            </BarDropdownItem>
                                            <BarDropdownItem To="/networkentityformat">
                                                Inquiry
                                            </BarDropdownItem>
                                        </BarDropdownMenu>
                                    </BarDropdown>
                                </BarDropdownMenu>
                            </BarDropdown>
                        </BarItem>

**Reply by stsrki** [reply]
Ah, you want to increase the size of the Bar itself? I thought you meant the text-overflow was the problem.

You can increase the size with theme options. Add this into `App.razor`.

```cshtml
<Blazorise.ThemeProvider Theme="@theme">
    <Router AppAssembly="typeof(App).Assembly">
        <Found Context="routeData">
            <RouteView RouteData="routeData" DefaultLayout="typeof(MainLayout)" />
        </Found>
        <NotFound>
            <p>Sorry, there's nothing at this address.</p>
        </NotFound>
    </Router>
</Blazorise.ThemeProvider>
@code{
    private Theme theme = new Theme
    {
        BarOptions = new ThemeBarOptions
        {
            HorizontalHeight = "64px",
            VerticalWidth = "350px"
        },
    };
}

```

**Reply by fatalkam** [reply]
Thank you not sure how I missed that but it worked perfectly.
